sync:
  image: n3llyb0y/wait
  links:
    - auth
    - postgres
  environment:
    PORTS: "9497 5432"
    # TIMEOUT: "${INTEGRATION_TEST_TIMEOUT}"
auth:
   image: wazopbx/wazo-auth-tests
   environment:
    - "TZ=UTC+1"
    - "XIVO_UUID=the-predefined-xivo-uuid"
   ports:
     - "9497"
   volumes:
     - "../../..:/usr/src/wazo-auth"
     - "./etc/wazo-auth/config.yml:/etc/wazo-auth/config.yml"
     - "${LOCAL_GIT_REPOS}/xivo-lib-python/xivo:/usr/local/lib/python2.7/site-packages/xivo"
   links:
     - "postgres"
   command: "wazo-auth"
postgres:
  image: wazopbx/wazo-auth-db
  ports:
    - "5432"
